example/reagent-demo/src/acme/frontend/drop_in.cljc

```diff
@@ -0,0 +1,8 @@
+(ns acme.frontend.drop-in
+  (:require [girouette.core :refer [css]]))
+
+;; This proves that girouette
+;; can work as a drop-in replacement for tailwind.
```

Comment:
Tailwind is a JS library and JS does not have symbols.

Please explain exactly how it is a drop-in replacement for Tailwind, it is not clear to me.